software/database/src/main/java/org/apache/brooklyn/entity/database/mysql/MySqlClusterImpl.java

```diff
@@ -119,7 +121,7 @@ protected void initEnrichers() {
         enrichers().add(Enrichers.builder()
                 .aggregating(MySqlNode.DATASTORE_URL)
                 .publishing(SLAVE_DATASTORE_URL_LIST)
-                .computing((Function)Functions.identity())
+                .computing((Function<Collection<String>, List<String>>)(Function)Functions.identity())
```

Comment:
Can't do the cast directly because the types are incompatible, first need to erase the generics and then do the cast.